pypa/pypa.go

```diff
@@ -0,0 +1,117 @@
+package pypa
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/aquasecurity/vuln-list-update/types"
+	"github.com/aquasecurity/vuln-list-update/utils"
+	"github.com/cheggaaa/pb"
+	"github.com/spf13/afero"
+	"golang.org/x/xerrors"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	pypaDir            = "pypa"
+	securityTrackerURL = "https://github.com/pypa/advisory-db/archive/refs/heads/main.zip"
```

Comment:
why not using https://osv-vulnerabilities.storage.googleapis.com/PyPI/all.zip ?
https://github.com/google/osv/blob/master/README.md#current-data-sources

Reply:
@AndreyLevchenko Could you check if we can use the above URL?

Reply:
@knqyf263 I think I can use the url above, but I need to understand benefits better. For now it looks pretty same to me.

Reply:
The benefit is we can have a single implementation for some languages (Python, Go and Rust).